assets/js/search.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * To perform client side searching. 
+ */
+(function () {
+    $('#search-box').keyup(function(){
+        var searchQuery = $(this).val();
+        
+        if(searchQuery.length === 0)  {
+            $('#filtered-records').html('');
+            $('#listings').show();
+            return;
+        } else {
+            // Hide the initial data before search began
+            $('#listings').hide();
+
+            var regex = new RegExp(searchQuery, "i");
```

Comment:
Using user-provided data like this, unsanitized, is a bit risky. Why do we need to use RegExp here? If you lower case the data and the query, an includes check should do it, right?

Reply:
True we do not need regex at all. This file has now been modified